src/Editor.tsx

```diff
@@ -83,8 +83,10 @@ const Editor = ({
   useEffect(() => {
     editor.setEditable(isEditable);
 
+    setIsEditable(isEditable);
+
     if (locale) i18n.changeLanguage(locale);
-  }, []);
+  }, [isEditable]);
```

Comment:
Without adding `isEditable` to the `useEffect` array, the editor doesn't respect changes to the Editor.tsx#isEditable property from external consumers.